src/components/Pokemon/PokemonCard.js

```diff
@@ -0,0 +1,53 @@
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+import './PokemonCard.css';
+
+function PokemonCard({endpoint}) {
+    const [pokemon, setPokemon] = useState({});
```

Comment:
Hier ook, dan kan je check verderop ook simpeler
```suggestion
    const [pokemon, setPokemon] = useState();
```